main_test.go

```diff
@@ -0,0 +1,60 @@
+// Copyright 2021 Northern.tech AS
+//
+//    Licensed under the Apache License, Version 2.0 (the "License");
+//    you may not use this file except in compliance with the License.
+//    You may obtain a copy of the License at
+//
+//        http://www.apache.org/licenses/LICENSE-2.0
+//
+//    Unless required by applicable law or agreed to in writing, software
+//    distributed under the License is distributed on an "AS IS" BASIS,
+//    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//    See the License for the specific language governing permissions and
+//    limitations under the License.
+package main
+
+import (
+	"flag"
+	"os"
+	"strings"
+	"testing"
+
+	"github.com/mendersoftware/mender-cli/cmd"
+)
+
+var runAcceptanceTests bool
+
+// used for parsing '-cli-args' for urfave/cli when running acceptance tests
+// this is because of a conflict between urfave/cli and regular go flags required for testing (can't mix the two)
+var cliArgsRaw string
+
+var _ = func() bool {
```

Comment:
Question:
Why does it need to return anything?

Reply:
I have no idea ([reference](https://github.com/mendersoftware/inventory/blob/master/main_test.go#L30-L33)). I'll ask to backend folks.

Reply:
I didn't get response. I removed it and still works... so :man_shrugging:

Reply:
Working code we like!